protocol/message.go

```diff
@@ -0,0 +1,40 @@
+package protocol
+
+// frontend message types
+const (
+	Query     = 'Q'
+	Terminate = 'X'
+	Parse     = 'P'
+	Bind      = 'B'
+	Describe  = 'D'
+	Execute   = 'E'
+	Sync      = 'S'
+)
+
+// Message is just an alias for a slice of bytes that exposes common operations on
+// Postgres' client-server protocol messages.
+// see: https://www.postgresql.org/docs/9.2/static/protocol-message-formats.html
```

Comment:
I know it's not yours, but can you please change it to:
```suggestion
// see: https://www.postgresql.org/docs/current/protocol-message-formats.html
```